### ISSUE TITLE
Unable to match CBL properly

### ISSUE THREAD

**Issue by rad96** [Kareadita/Kavita#3243]
### What happened?

This CBL contains - https://github.com/DieselTech/CBL-ReadingLists/blob/main/Marvel/Master%20Reading%20Order/CBRO%20-%20With%20Events/%5BMarvel%5D%20All-New%2C%20All%20Different%20Marvel-%20All-New%2C%20All-Different%20Marvel%20Part%20%231%20(WEB-CBRO).cbl
This: https://comicvine.gamespot.com/deadpool-the-mercs-for-money/4050-87793/
And this: https://comicvine.gamespot.com/deadpool-the-mercs-for-money/4050-92360/
They result, due to same volume start year, in same series
Kavita do this:
![изображение](https://github.com/user-attachments/assets/80ad4eed-6437-498a-9e67-4b3c65d19706)
Which is merging comics under same issue number

Because of that I can't match CBL properly

### What did you expect?

Komga do this
![изображение](https://github.com/user-attachments/assets/daa16947-944f-484f-99da-4d56e305772f)
Simply NOT merging books under same issue number
And after that it allows you to at least manually match books properly in CBL import

I expect an ability to import such CBL properly

### Kavita Version Number - If you don not see your version number listed, please update Kavita and see if your issue still persists.

0.8.3 - Stable

### What operating system is Kavita being hosted from?

Windows

### If the issue is being seen on Desktop, what OS are you running where you see the issue?

None

### If the issue is being seen on Mobile, what OS are you running where you see the issue?

None

**Comment by therobbiedavis** [comment]
The issue here I think is that Kavita considers these two series the same series because the name and start year match, therefore merges the issues.

There may be a cultural or language barrier but I'll just say please keep any future bug reports succinct and free from emotion.

**Comment by rad96** [comment]
> The issue here I think is that Kavita considers these two series the same series because the name and start year match, therefore merges the issues.
> 
> There may be a cultural or language barrier but I'll just say please keep any future bug reports succinct and free from emotion.

Yeah, not much emotion I had there, just was answering literal question from issue creation form:
![изображение](https://github.com/user-attachments/assets/80f18a22-a7d7-4734-bc59-ad895c26fcb5)

Changed a bit of probably bad interpreted line
I meant I have not much of an idea how to do that properly, but at the end I want to have matched CBL

**Comment by Fletcher8788** [comment]
I am experiencing the same issue on 0.8.9.25 and on 0.8.9.1. With series like the above and https://comicvine.gamespot.com/x-23-1-one-shot/4000-201027/ https://comicvine.gamespot.com/x-23-1-the-killing-dream-part-1/4000-234641/. I have ensured both files are in different folders and renamed the files so that they have different names but the same issue occurs. In .1 the one shot is automatically imported rather than the series and in .25 the import fails saying it cannot find 234641

**Comment by majora2007** [comment]
> I am experiencing the same issue on 0.8.9.25 and on 0.8.9.1. With series like the above and https://comicvine.gamespot.com/x-23-1-one-shot/4000-201027/ https://comicvine.gamespot.com/x-23-1-the-killing-dream-part-1/4000-234641/. I have ensured both files are in different folders and renamed the files so that they have different names but the same issue occurs. In .1 the one shot is automatically imported rather than the series and in .25 the import fails saying it cannot find 234641

Please don't put nightly release issues on non-nightly issues. Please create a new issue since this issue is more about how Kavita organizes vs the actual CBL mechanic.

**Comment by Fletcher8788** [comment]
Sure Ill log a separate issue for it on 0.8.9.25
